site/en/docs/devtools/recorder/index.md

```diff
@@ -105,27 +115,57 @@ Let's walk through the options to edit the user flows.
-    - For example, if the **Recorder** successfully queries the element with the first selector, it will skip the second selector and proceed to the next step.
-3. You can add or remove any selectors. For example, you can remove the *selector #2* because just `aria/Cappucino` is sufficient in this case. Hover over the *selector #2* and click on **-** to remove it.
-    {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/qEat3jT1Og68urhmSkef.png", alt="The DevTools recorder panel shows an option to remove a selector", width="800", height="663" %}
-4. The **selector** is editable too. For example, if you want to select *Mocha* instead of *Cappucino*, you can edit the selector value to *aria/Mocha* instead.
-  {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/OdihJ4BROOXZ93eidr4b.png", alt="edit a selector", width="800", height="663" %} 
-5. [Replay](#replay) the flow now, it should select *Mocha* instead of *Cappucino*.
-6. Try to edit other step properties such as **type**, **target**, **value** and more.
+### Expand steps {: #expand-step }
+
+1. Expand each step to see the details of the action. For example, expand the *Click Element "Cappuccino"* step.
+  {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/WUIZb8eMWfcPZHPyQ5C5.png", alt="In the recorder panel, the Cappuccino element has been expanded to reveal type, target, selectors, offset X, and offset Y.", width="800", height="773" %}
+1. The step above shows two **selectors**. For more information, see [Selector types](#selector-types).
+   When replaying the user flow, the **Recorder** tries to query the element with one of the selectors by sequence.
+   For example, if the **Recorder** successfully queries the element with the first selector, it will skip the second selector and proceed to the next step.
+1. You can add or remove any selectors. For example, you can remove the *selector #2* because just `aria/Cappuccino` is sufficient in this case. Hover over the *selector #2* and click on **-** to remove it.
+    {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/ivZru7CFneS6fu5BAcIJ.png", alt="The DevTools recorder panel shows an option to remove a selector", width="800", height="758" %}
+1. The **selector** is editable too. For example, if you want to select *Mocha* instead of *Cappuccino*, you can edit the selector value to *aria/Mocha* instead.
+  {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/OdihJ4BROOXZ93eidr4b.png", alt="edit a selector", width="800", height="663" %}
+   Alternatively, click the **Select**{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/ihbO22H4EA0HfrNX6gV1.png", alt="Select button", width="24", height="22" %} button and then click *Mocha* on the page.
+1. [Replay](#replay) the flow now, it should select *Mocha* instead of *Cappuccino*.
+1. Try to edit other step properties such as **type**, **target**, **value** and more.
+
+#### Selector types {: #selector-types }
```

Comment:
How about move this section to the bottom, create a ## Selectors section, with both this and  the customize the recording's selector section?

1. Both of this section are currently too far away from each other, ideally we put them together.
2. Selector isn't belong the edit steps section